protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1687,9 +1687,12 @@ message ListTransfersRequest {
   // Restrict transfer to those where the given public key is a sender or receiver.
   optional string pubkey = 1;
   // Restrict transfers to those in the given direction from the supplied public key.
+  // When is_reward is true, and pubkey is set, then direction MUST be TRANSFER_DIRECTION_DIRECTION_FROM.
```

Comment:
```suggestion
  // When is_reward is true, and pubkey is set, then direction MUST be 'from'.
```

Reply:
Doesn't it make more sense to use the values that make sense within the types in the proto files? `'from'` is not a valid `TransferDirection` value after all

Reply:
Are they exactly the same for REST and gRPC on the user side?

Reply:
Yes, looking at the `trading_data.swagger.json` file:

```json
          {
            "name": "direction",
            "description": "Restrict transfers to those in the given direction from the supplied public key.",
            "in": "query",
            "required": false,
            "type": "string",
            "enum": [
              "TRANSFER_DIRECTION_UNSPECIFIED",
              "TRANSFER_DIRECTION_TRANSFER_FROM",
              "TRANSFER_DIRECTION_TRANSFER_TO",
              "TRANSFER_DIRECTION_TRANSFER_TO_OR_FROM"
            ],
            "default": "TRANSFER_DIRECTION_UNSPECIFIED"
          }
```